src/index.js

```diff
@@ -92,7 +92,7 @@ Koop.prototype.register = function (plugin, options) {
     case 'provider':
       return this._registerProvider(plugin, options)
     case 'cache':
-      return this._registerCache(plugin)
+      return this._registerCache(plugin, options)
     case 'plugin':
       return this._registerPlugin(plugin)
```

Comment:
lets also adjust this one for consistency

Reply:
do you mean adjust the `_registerPlugin` method below it?

Reply:
yes

Reply:
ok, any of the other ones below it?

Reply:
Yeah, just add the options arg to all